README.md

```diff
@@ -18,19 +18,232 @@ Alternatively, you can use the Open in Colab browser extension ([Chrome](https:/
+| output        | jsonb             | The output of the check.                                                                                                     |
+| collection_id | integer           | The collection to which this check result relates.                                                                           |
+
+#### run_collection
+
+| Column        | Type              | Description                               |
+| ------------- | ----------------- | ----------------------------------------- |
+| run_id        | character varying | A run of data quality checks.             |
+| collection_id | integer           | A collection that was checked in the run. |
+| id            | integer           | A unique identifier for the relationship. |
+
+#### indicator_coverage
+        
+| Column           | Type              | Description                                                   |
+| ---------------- | ----------------- | ------------------------------------------------------------- |
+| run_id           | character varying | The run in which this coverage result was calculated.         |
+| collection_id    | integer           | The collection to which this coverage result relates.         |
+| indicator_source | character varying | The source from which the indicator was drawn.                |
+| indicator        | character varying | The indicator to which this coverage result relates.          |
+| fields           | jsonb             | The fields needed to calculate the indicator.                 |
+| successes        | numeric           | A count of objects for which the indicator can be calculated. |
+| checks           | numeric           | The number of checks performed.                               |
+
+#### oc4ids_schema
+
+| Column           | Type | Description                                                                                                                                                                                                                                             |
+| ---------------- | ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| section          | text | The first part of the JSON path to the field in the data, e.g. budget.                                                                                                                                                                                  |
+| path             | text | The JSON path to the field in the data, e.g. budget/description.                                                                                                                                                                                        |
```

Comment:
Are backticks required for field paths?

Reply:
This is an example of a possible value of the `path` column. The column's values don't include backticks so, since the column descriptions are stored as PostgreSQL comments and are expected to be read in tools that don't render Markdown, I think it's better to omit the backticks here. Otherwise, users might expect the column's values to include backticks.